spec/controllers/queue_items_controller_spec.rb

```diff
@@ -82,11 +89,71 @@
         expect(user1.queue_items.size).to eq(0)
       end
 
-      it "redirect to queue index" do
+      it "should redirect to queue index" do
         delete :destroy, user_id: user1, id: queue_item2.id
         expect(response).to redirect_to user_queue_items_path(user1)
       end
 
+      it "should reorder position numbers" do
+        user1.queue_items << queue_item1
+        user1.queue_items << queue_item2
+        user1.queue_items << queue_item3
```

Comment:
Adding `queue_item1 - 3` to the collection of queue_items of `user1` seems to be unnecessary, because 

```ruby
let(:queue_item2) { Fabricate(:queue_item, user: user1, position: 2, id: 2) }
let(:queue_item3) { Fabricate(:queue_item, user: user1, position: 3, id: 3) }
```